src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -191,3 +210,16 @@ func (q *qemuAmd64) appendImage(ctx context.Context, devices []govmmQemu.Device,
 func (q *qemuAmd64) appendBridges(devices []govmmQemu.Device) []govmmQemu.Device {
 	return genericAppendBridges(devices, q.Bridges, q.qemuMachine.Type)
 }
+
+func (q *qemuAmd64) appendMemEncrypt(devices []govmmQemu.Device) []govmmQemu.Device {
+	object := govmmQemu.Object{
+		Type: govmmQemu.SevGuest,
+		ID:   "sev0",
+		CBitPos: cpuid.MemEncrypt.CBitPosition,
+		ReducedPhysBits: cpuid.MemEncrypt.PhysAddrReduction,
+	}
+
+	devices = append(devices, object)
+
+	return devices
+}
```

Comment:
This would be handled in a more platform agnostic way by `appendProtectionDevice()` in #1590